test-common/src/main/scala/io/lenses/streamreactor/connect/testcontainers/connect/ConfigProperty.scala

```diff
@@ -0,0 +1,21 @@
+package io.lenses.streamreactor.connect.testcontainers.connect
+
+trait CnfVal[T <: Any] {
```

Comment:
I suppose these are meant to ensure the exact type in configuration. Then the best practice is to use:

```
case class WrappedValue[T](val underlying: T) extends AnyVal 
```

This does not do a runtime allocation it just replaces the value. Anda voids the virtual method call.

Reply:
ok, replaced - thanks.